scripts/proposals/veANGLE/KillveANGLE.s.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.19;
+
+import { console } from "forge-std/console.sol";
+import { Wrapper } from "../Wrapper.s.sol";
+import "../../Constants.s.sol";
+import { ProxyAdmin } from "oz/proxy/transparent/ProxyAdmin.sol";
+
+contract KillveANGLE is Wrapper {
+    SubCall[] private subCalls;
+    uint256 public constant chainId = CHAIN_ETHEREUM;
+
+    function run() external {
+        vm.selectFork(forkIdentifier[chainId]);
+        address veANGLE = 0x0C462Dbb9EC8cD1630f1728B2CFD2769d09f0dd5;
+        address timelock = 0x09D81464c7293C774203E46E3C921559c8E9D53f;
+        address proposalSender = 0x896D64B4B7265273dDCD00808f3579563f9790A8;
+        address onchainGovernorContract = 0x748bA9Cd5a5DDba5ABA70a4aC861b2413dCa4436;
+        address govMultisig = 0xdC4e6DFe07EFCa50a197DF15D9200883eF4Eb1c8;
+
+        // Activate veANGLE emergency withdrawal
+        subCalls.push(SubCall(chainId, veANGLE, 0, abi.encodeWithSelector(IVeAngle.set_emergency_withdrawal.selector)));
```

Comment:
worth doing this at the very end, just to be extra sure all are passing and not just this one